### PR TITLE
Changes to be able to build as Java 11 project

### PR DESCRIPTION
**Pull request by obfischer**
Changes to the project to be able to run it as Java 11 application.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -342,6 +369,33 @@
...
+                            </goals>
+                            <configuration>
+                                <rules>
+                                    <requireJavaVersion>
+                                        <version>[11,)</version>
+                                    </requireJavaVersion>
+                                </rules>
+                            </configuration>
+                        </execution>
+                        <execution>
```
Comment (StephanPirnbaum) [inline pom.xml]: Since no dependencies are configured here, we can drop this configuration

File: pom.xml [hunk 2]
```diff
@@ -562,10 +625,10 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-compiler-plugin</artifactId>
                         <configuration>
-                            <source>1.8</source>
-                            <target>1.8</target>
-                            <testSource>1.8</testSource>
-                            <testTarget>1.8</testTarget>
+                            <source>11</source>
```
Comment (DirkMahler) [inline pom.xml]: This configuration already exists in lines 242ff

**Review by DirkMahler — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (obfischer) [reply pom.xml]: You are right. I will remove it. Do we still need the profile itself?

File: pom.xml (hunk 1 above)
Comment (StephanPirnbaum) [reply pom.xml]: execution id: enforce-banned-dependencies

**Review by StephanPirnbaum — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (obfischer) [reply pom.xml]: Removed, commited and pushed.

**Review by StephanPirnbaum — APPROVED** [review]

**Review by DirkMahler — APPROVED** [review]

**Pull request merged**